README.md

```diff
@@ -212,49 +188,96 @@ inputs = generate_random_inputs(onnx_filepath, batch_size)
 engine = compile_model(onnx_filepath, batch_size)
 outputs = engine.run(inputs)
 ```
-**Compatibility/Support Notes**
+Compatibility/Support Notes
 - ONNX version 1.5-1.7
 - ONNX opset version 11+
 - ONNX IR version has not been tested at this time
 
-For a more in-depth read on available APIs and workflows, check out the [examples](https://github.com/neuralmagic/deepsparse/blob/main/examples/) and [DeepSparse Engine documentation.](https://docs.neuralmagic.com/deepsparse)
+The [GitHub repository](https://github.com/neuralmagic/deepsparse) includes package APIs along with examples to quickly get started benchmarking and inferencing sparse models.
+
+__ __
+
+## Scheduling Single-Stream, Multi-Stream, and Elastic Inference
+
+The DeepSparse Engine offers up to three types of inferences based on your use case. You can read more details here: [Inference Types](https://github.com/neuralmagic/deepsparse/blob/main/docs/source/scheduler.md).
```

Comment:
```suggestion
The DeepSparse Engine offers up to three types of inferences based on your use case. Read more details here: [Inference Types](https://github.com/neuralmagic/deepsparse/blob/main/docs/source/scheduler.md).
```

Reply:
When possible or relevant, try to avoid using the phrase "You can ...." in technical writing. There's no need to offer that since the reader will decide on their own if they will or will not do the action.